src/global/globalSlice.ts

```diff
@@ -43,3 +51,24 @@ export const updateBreakpointWidth: ActionCreator<
       dispatch(setBreakpointWidth(viewportWidth));
     }
   };
+
+const globalSlice = createSlice({
+  name: 'global',
+  initialState: defaultState as GlobalState,
```

Comment:
`defaultState` already knows that it's of type `GlobalState`

Reply:
Removed the casting